### PR TITLE
add hermez to l2

### PR DESCRIPTION
**Pull request by corwintines**
## Description
- Add Polygon Hermez to layer 2

### CODE REVIEW

File: src/data/layer-2/layer-2.json [hunk 1]
```diff
@@ -158,6 +158,22 @@
...
+      "developerDocs": "https://docs.hermez.io/",
+      "l2beat": "https://l2beat.com/projects/hermez/",
+      "bridge": "https://wallet.hermez.io/login",
+      "bridgeWallets": ["MetaMask", "WalletConnect"],
+      "blockExplorer": "explorer.hermez.io",
+      "ecosystemPortal": "",
+      "tokenLists": "",
+      "noteKey": "",
+      "purpose": ["payments"],
+      "description": "Polygon Hermez is an open-source ZK-Rollup optimised for secure, low-cost and usable token transfers on the wings of Ethereum",
```
Comment (samajammin) [inline src/data/layer-2/layer-2.json:174]:
```suggestion
      "description": "Polygon Hermez is an open-source ZK-Rollup optimized for secure, low-cost and usable token transfers on the wings of Ethereum",
```

Comment (samajammin) [reply src/data/layer-2/layer-2.json:174]: https://ethereum.org/en/contributing/style-guide/#american-english

**Comment by samajammin** [comment]
LGTM, assuming it meets the [criteria of the listing policy](https://ethereum.org/en/contributing/adding-layer-2/). Have we confirmed?

**Comment by corwintines** [comment]
Closing while I create an issue and do a bit more research, will reopen after discussion there.

**Comment by Carrussalgreencompany** [comment]
Good user experience.

**Pull request closed without merging**